### DISCUSSION TITLE
Strange behaviour

### DISCUSSION THREAD

**Discussion by QServ** [Koenkk/zigbee2mqtt#5267] [General]
I have a strange problem.
When I publish a command to one specific group another group reacts. The log logs the MQTT topics and commands and it looks correct. 
Is there a way to get what is sent to the zigbee network into the log also?
I have tried to set log_level to debug but It still just logs the MQTT publishing.

**Comment by spantobi** [comment]
I have the same problem. I've been trying to fix this for days now. I turn on a group of 4 lights in my laundry room but only 2 of them turn on but then another 2 in the hallway on the first floor also turn on. Not sure how to fix it. It looks correct when I check the groups.
